lmnet/configs/example/object_detection.py

```diff
@@ -15,7 +15,6 @@
 # =============================================================================
 from easydict import EasyDict
 import tensorflow as tf
-
```

Comment:
Please don't delete line.
See also
https://www.python.org/dev/peps/pep-0008/#id23

Reply:
I've fixed it